pages/common/nix-env.md

```diff
@@ -30,3 +30,7 @@
 - Upgrade all packages:
 
 `nix-env -u`
+
+- Install a package from a URL:
+
+`nix-env -i {{pkg_name}} --file {{URL}}`
```

Comment:
Perhaps we could have a concrete example of a URL?

If the domain varies, perhaps `example.com` would be a good idea.

Reply:
The domain name does vary. I'll use `example.com`.

Reply:
@sbrl, fixed.